### PR TITLE
Adding Explosive Shovel and beautifying Tech Category.

### PR DESCRIPTION
**Pull request by LinoxGH**
- Adding a new Tool called Explosive Shovel that breaks a 9x9x9 area of shovel blocks.
- Moving the Hologram Projector in Tech category a little bit upwards to make all multitools and jetboots in the same line and under/above their relevant items.(f.e lvl 1 jetboot, multitool and jetpack are now in the same column.)

### CODE REVIEW

File: src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(98, "Advanced Mining 101", 42), SlimefunItems.ADVANCED_DIGITAL_MINER);
 	    Slimefun.registerResearch(new Research(99, "Composting Dirt", 3), SlimefunItems.COMPOSTER);
 	    Slimefun.registerResearch(new Research(100, "Farmer Shoes", 4), SlimefunItems.FARMER_SHOES);
-	    Slimefun.registerResearch(new Research(101, "Explosive Pickaxe", 28), SlimefunItems.EXPLOSIVE_PICKAXE);
+	    Slimefun.registerResearch(new Research(101, "Maximum Explosivity", 30), SlimefunItems.EXPLOSIVE_PICKAXE, SlimefunItems.EXPLOSIVE_SHOVEL);
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Rather just call it "Explosive Tools", this sounds a bit exaggerated.

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java [hunk 2]
```diff
@@ -1756,6 +1794,56 @@ else if (b.getType().name().endsWith("_SHULKER_BOX")) {
...
+			public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List<ItemStack> drops) {
+				if (SlimefunManager.isItemSimiliar(item, SlimefunItems.EXPLOSIVE_SHOVEL, true)) {
+					e.getBlock().getWorld().createExplosion(e.getBlock().getLocation(), 0.0F);
+					e.getBlock().getWorld().playSound(e.getBlock().getLocation(), Sound.ENTITY_GENERIC_EXPLODE, 1F, 1F);
+					for (int x = -1; x <= 1; x++) {
+						for (int y = -1; y <= 1; y++) {
+							for (int z = -1; z <= 1; z++) {
+								Block b = e.getBlock().getRelative(x, y, z);
+								boolean correctType = false;
+								for (Material mat : MaterialCollections.getAllFishItems()) {
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java]: getAllFishItems ????

File: src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java (hunk 1 above)
Comment (LinoxGH) [reply src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java]: Alright I'll fix it now and also the placeholder method in SlimefunSetup

File: src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java (hunk 2 above)
Comment (LinoxGH) [reply src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java]: That was a placeholder method so I could build the plugin to test if setup and etc. works.

Comment (TheBusyBiscuit) [reply src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java]: Well, use CS-CoreLib2's method then.

All you have to do is update the pom.xml and change the version of CS-CoreLib to 0.3.8
And add the path of the materials package in the shade-plugin section

**Comment by LinoxGH** [comment]
Did the requested changes.

**Comment by TheBusyBiscuit** [comment]
You still have to add the materials package to the "maven-shade-plugin" section of the pom.xml

**Comment by LinoxGH** [comment]
Added it although I'm not 100% sure if I did it right.

**Pull request merged**